File: codeforces/virtual/372/C.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    long long n; cin >> n;

    long long i = 1;
    long long cur = 2;
    while (n--) {
        long long mult = i*(i+1);
        if (mult>cur)
            cout << "nigggaaa " << i << endl;
        cout << (mult*mult - cur)/i << endl;
        cur = mult;
        i++;
    }

    return 0;
}

File: codeforces/virtual/372/C.java
import java.util.*;
import java.math.BigInteger;

public class ProblemC {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();

        BigInteger i = BigInteger.ONE;
        BigInteger cur = BigInteger("2");
        while (i<=n) {
            BigInteger mult = i.multiply(i.add(BigInteger.ONE));
            BigInteger ans = mult.multiply(mult).subtract(cur).divide(i);
            System.out.println(ans);
            cur = mult;
            i = i.add(BigInteger.ONE);
        }
    }
}
